supportedBackends/google/pipelines/v2beta/src/main/scala/cromwell/backend/google/pipelines/v2beta/PipelinesUtilityConversions.scala

```diff
@@ -67,7 +68,7 @@ trait PipelinesUtilityConversions {
     // characters (typically emoji). Some databases have trouble storing these; replace them with the standard
     // "unknown character" unicode symbol.
     val name = Option(event.getContainerStopped) match {
-      case Some(_) => cleanUtf8mb4(event.getDescription)
+      case Some(_) => StringUtil.cleanUtf8mb4(event.getDescription)
```

Comment:
Can we use the config to include this metadata in what we're sanitizing rather than cleaning it here?

Reply:
Followup: in theory we could, but we would need to check every single `executionEvent`, of which there are many, because their keys are all like:
```
executionEvents[1173839490]:startTime
executionEvents[1173839490]:endTime
executionEvents[1173839490]:grouping
executionEvents[1173839490]:description
```
Leaving this comment in case anyone else has this idea, but I'm OK leaving this as-is for now. Would be great to add a comment explaining this context, though (we sanitize other metadata values elsewhere, but are handing this one differently). 

We'll also need to keep in mind that we may need to do something different for Batch. I'll create a ticket in that epic.